### PR TITLE
Update scannermsbuild.properties for 5.14

### PR DESCRIPTION
**Pull request by tom-howlett-sonarsource**

### CODE REVIEW

File: scannermsbuild.properties [hunk 1]
```diff
@@ -16,6 +16,15 @@ flavors.core20.label=.NET Core 2.1
 flavors.core31.label=.NET Core 3.1
 flavors.framework46.label=.NET Framework 4.6+
 
+5.14.description=Support SonarQube 10.3 API
```
Comment (christophelevis) [inline scannermsbuild.properties:19]: I don't think this is a change related to 10.3. The drop should actually happen in 10.4 ([SONAR-19233](https://sonarsource.atlassian.net/browse/SONAR-19233)). Should we simply mention "Support for future versions of SonarQube"?

Comment (tom-howlett-sonarsource) [reply scannermsbuild.properties]: I updated, I think its good to mention its 10.4 related

File: scannermsbuild.properties [hunk 2]
```diff
@@ -16,6 +16,15 @@ flavors.core20.label=.NET Core 2.1
 flavors.core31.label=.NET Core 3.1
 flavors.framework46.label=.NET Framework 4.6+
 
+5.14.description=Support SonarQube 10.4 API changes
```
Comment (christophelevis) [inline scannermsbuild.properties:19]: Should we use the word "Prepare..." since we are in advance?

**Review by christophelevis — APPROVED** [review]
LGTM. Just made a suggestion.

**Pull request merged**